doc/modelling/binding/multi_component_bi_langmuir_ldf.rst

```diff
@@ -0,0 +1,22 @@
+.. _multi_component_bi_langmuir_model_ldf:
+
+Multi Component Bi-Langmuir LDF
```

Comment:
I find the text hard to understand. I know it's based on the original Bi-Langmuir text which could also require some more depth but it's not clear to me (from the text alone) what the LDF does to the model. I suggest to reference the Bi-Langmuir documentation and only describe the LDF approach.

Reply:
I have described the LDF approach in the[ intro section ](doc/modelling/binding/index.rst) of binding models, and while writing about the actual isotherm I am just referring the approach. My query here is, what exactly should I describe about LDF approach here?